Data/Entities/Pet.cs

```diff
@@ -7,7 +7,7 @@ namespace HelloPets.Data.Entities;
 public class Pet : Entity
 {
     public Nickname Nickname { get; private set; } = null!;
-    public string TutorId { get; private set; }
+    public string TutorId { get; private set; } = null!;
     public Tutor Tutor { get; private set; } = null!;
```

Comment:
esqueci de mencionar isso no review anterior, sempre que for vincular um objeto de relacionamento, adiciona a keyworkd `virtual` na frente do tipo:
`public virtual Tutor`

Mesma coisa pra pets

dessa forma o EF reconhece certo que sao entidades conectadas e nao da problema com tipos nullable